packages/ownable/src/lib.rs

```diff
@@ -307,11 +296,7 @@ mod tests {
     use super::*;
 
     fn mock_addresses() -> [Addr; 3] {
-        [
-            Addr::unchecked("larry"),
-            Addr::unchecked("jake"),
-            Addr::unchecked("pumpkin"),
-        ]
+        [Addr::unchecked("larry"), Addr::unchecked("jake"), Addr::unchecked("pumpkin")]
```

Comment:
Can you revert the formatting here?

Otherwise, this PR looks good.

Reply:
Reverted